system-tests/projects/v8-snapshot/example-express/snapshot/snapshot.js

```diff
@@ -0,0 +1,4 @@
+module.exports = { express: require('express') }
+// module.exports = { depd: require('depd') }
+// const deprecate = require('depd')('body-parser')
+// module.exports = { deprecate }
```

Comment:
do these serve a purpose for this test?

```suggestion
```

Reply:
Updated here: [`9ac5768` (#23566)](https://github.com/cypress-io/cypress/pull/23566/commits/9ac57680068b11b55ee9c565d3df2f0c80892f0a)